src/main/java/com/mattmalec/pterodactyl4j/utils/Checks.java

```diff
@@ -59,4 +60,11 @@ public static void notEmpty(Collection<?> collection, String name) {
 			throw new IllegalArgumentException(name + " cannot be empty!");
 		}
 	}
+
+	public static void validateUrl(String url, String name) {
```

Comment:
i would preferably avoid regex when possible. perhaps this could work?

```suggestion
	public static void validateUrl(String url, String name) {
	notNull(url, name);
	try {
            new URL(url);
        } catch (MalformedURLException e) {
            throw new IllegalArgumentException(name + " must be a valid URL!");
        }
}
```

Reply:
I was originally planned to do like this, but I though you would prefer to use a Regex, to me it's all good!